tasks/dataset-split/task.yaml

```diff
@@ -0,0 +1,25 @@
+id: split-dataset
+name: split-dataset
+description: Task to split a dataset into a user-defined number of parts
+is_active: true
+project_type: 8
+param_groups:
+  - name: inputs
+    params:
+      - name: dataset
+        value: null
+        required: true
+        data_type: dataset
+        description: Dataset which is going to be split into multiple parts
+      - name: datasetParts
+        value: 2
```

Comment:
Is 2 default and maximum number of parts dataset can be split to? if yes than remove property from .yaml file and if not set the default value to null like for the other properties.